MetaMask/metamask-extension
# Part of #18714: Replacing deprecated constants with enums

## Explanation

- This PR is a part of #18714 
- Made changes to 
- `ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.js`
- `ui/components/app/modals/contract-details-modal/contract-details-modal.js`
- `ui/components/app/network-account-balance-header/network-account-balance-header.js`



## Screenshots/Screencaps



### Before


### Theres no story for `advanced-gas-fee-defaults`
`contract-details-modal`
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/b9f495ce-a4be-4645-9302-c732c05525b8)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/d2480f16-d96e-4e7b-8acb-4e398be4f156)

`network-account-balance-header`
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/8dff7622-cf66-4aa2-8745-51603c9f770f)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/904ef39e-95b1-4ba4-9493-415e33a71307)

### After


`contract-details-modal`
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/8b0e3655-bd8a-411f-a9fb-9c40bc488907)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/f9e3a142-996a-4a89-96a4-b0eb10df8edb)

`network-account-balance-header`
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/e548d27e-123f-4a0a-8c83-db87d59d0073)
![image](https://github.com/MetaMask/metamask-extension/assets/79097544/a8482000-2240-489c-aa67-1d510eadc1ff)

## Manual Testing Steps

- Pull this branch
- Run `yarn storybook`
- Check stories for `contract-details-modal` and `network-account-balance-header`



## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
